bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
Hello, @bowlofeggs !

I started asserting the proper behavior like such:
`result = cmd('')
self.assertEqual(result[0], 'output')
self.assertEqual(result[1], None)
self.assertEqual(result[2], 0)`

Then I realized checking the result of the cmd() function was pretty silly, because cmd() is returning stderr from the subprocess regardless. I need to check the messages at the INFO and ERROR levels of a logger. I have been searching online all day, but I still don't understand how to check the messages at specific levels of a logger... I found this tool: http://testfixtures.readthedocs.io/en/latest/logging.html, but I'm not sure whether using such a library is acceptable, or if there's a simpler way to check... Or maybe whether to only return the err from command when  Any guidance would be greatly appreciated!